packages/react/runtime/src/auto.ts

```diff
@@ -378,6 +379,12 @@ export function installJSXHooks() {
 }
 
 export function installAutoSignalTracking() {
+	const [major] = reactVersion.split(".").map(Number);
+	if (major >= 19) {
+		throw new Error(
+			"Automatic signals tracking is not supported in React 19 and later, try the Babel plugin instead https://github.com/preactjs/signals/tree/main/packages/react-transform#signals-react-transform."
```

Comment:
Might be good to shorten the URL.

Reply:
```suggestion
			"Automatic signals tracking is not supported in React 19 and later, try the Babel plugin instead https://bit.ly/3zbndcL."
```

Reply:
Bit late, but I think most would prefer a bare & long URL over a shortened link leading god only knows where -- link shorteners tend to scream "sketchy".

Reply:
😅 thought it was just me, I'll revert it